spec/features/auth_spec.rb

```diff
@@ -16,4 +16,17 @@
     expect(page).to have_content("user@example.com")
   end
 
+ scenario 'Users can register' do
+   visit root_path
+   click_on "Register"
+   fill_in "Name", with: "Ryan"
+   fill_in "Email", with: "ryan@email.com"
+   fill_in "Password", with: '12345'
+   fill_in "Confirm", with: '12345'
+   click_button "Register"
+   expect(page).to have_content("ryan@email.com")
+ end
```

Comment:
All of this indentation is off by precisely 1 space. Other than that this is great. For readability you may want to consider putting a blank line between all of the setup steps (like `click_button` etc) and the assertion.